doto-frontend/src/components/ModalContent.js

```diff
@@ -60,8 +62,8 @@ const ModalContent = props => {
         }
     };
 
-    const handleDurationChange = event => {
-        setSelectedDuration(event.target.value);
+    const handleDurationChange = date => {
+        setselectedDuration(date);
```

Comment:
You can probably simplify this by passing the hook setselectedDuration to the component directly instead of wrapping in this function :)